Apps/AdminWebClient/src/ClientApp/src/components/core/CommunicationTable.vue

```diff
@@ -228,11 +231,10 @@ export default class CommunicationTable extends Vue {
         }
     }
 
-    private editBanner(item: Communication) {
+    private edit(item: Communication) {
         this.editedIndex = this.communicationList.indexOf(item);
```

Comment:
Looks like the actual index does not matter. So how about making this a "isNewCommunication"?

Reply:
Good idea